testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -162,11 +164,22 @@ def test_calculate_mean(pca_aligned, u, u_aligned):
 
 
 def test_given_mean(pca_aligned, u):
-    pca = PCA(u, select=SELECTION, align=False,
-              mean=pca_aligned._mean).run()
+    # create dummy atomgroup to populate with the mean values
+    num_atoms = len(u.select_atoms(SELECTION))
+    ag = mda.Universe.empty(num_atoms, trajectory=True).select_atoms('all')
+    ag.positions = pca_aligned.mean.reshape((num_atoms, 3))
```

Comment:
according to the docs we should just be able to use `mean_atoms` back as the atomgroup with the mean positions right?

Reply:
From the current docs, yes - but, per discussion in #3285, it seems that `mean_atoms` does not actually contain the mean positions. If we fix that, then we could just use `mean_atoms` here;  I decided to do it roundabout like this for now with a view of getting the more 'urgent' changes merged rather than debating exactly what `mean_atoms` should be doing (imo the doc changes in #3285 reflect what `mean_atoms` seem to actually be storing atm so it's not a big after that goes through). 

All that being said, there is still the option to fix `mean_atoms` here (or in 3285), if that seems better? (Personally, it feels like both `means` and `mean_atoms` would be better as position arrays rather than atomgroups, but I'm not super familiar with pca and how people use this part of the input/output so others should weigh in first)

Reply:
I'd vote for just fixing it properly now (as a coordinate array) and not going through a deprecation cycle.

Reply:
Works for me.